core/src/main/java/com/github/mizool/core/converter/EnumConverter.java

```diff
@@ -40,4 +45,26 @@ public String fromPojo(Enum<?> enumValue)
         }
         return result;
     }
+
+    public List<String> fromPojos(List<Enum<?>> enumValues)
```

Comment:
this is redundant with `com.github.mizool.core.converter.Converters` introduced by PR #92. also, it does not comply with OMG-27 in that it passes `null` collections to the core layer.

Reply:
see also https://beanvalidation.org/proposals/BVAL-508/

Reply:
The calls to use the `Converters` class are
`Converters.fromPojos(foos, enumConverter::fromPojo)`
and
`Converters.toPojos(fooDtos, fooDto -> enumConverter.toPojo(Foo.class, fooDto))`
respectively.